71_RayTracingPipeline/app_resources/raytrace.rchit.hlsl

```diff
@@ -75,15 +78,15 @@ float3 calculateNormals(int primID, STriangleGeomInfo geom, float2 bary)
 void main(inout PrimaryPayload payload, in BuiltInTriangleIntersectionAttributes attribs)
 {
     const int primID = PrimitiveIndex();
```

Comment:
btw there's no new SPIR-V for this but existing SPIR-V gets modified 
<img width="1212" height="543" alt="image" src="https://github.com/user-attachments/assets/5ecb8406-8a76-44bf-9dca-8200aaff3921" />